### ISSUE TITLE
Place 3 buttons vertically

### ISSUE THREAD

**Issue by indapublic** [mmazzarolo/react-native-dialog#47]
Is it possible place three buttons vertically (like Alert.alert)?

**Comment by mmazzarolo** [comment]
@indapublic not yet... it's in the [TODO](https://github.com/mmazzarolo/react-native-dialog#todo) section of the README but I don't think I'll be able to look into it soon. 
Open to PRs!

**Comment by Desteel** [comment]
> @indapublic not yet... it's in the [TODO](https://github.com/mmazzarolo/react-native-dialog#todo) section of the README but I don't think I'll be able to look into it soon.
> Open to PRs!

Hello. 
Any news on this request?

**Comment by Desteel** [comment]
> Is it possible place three buttons vertically (like Alert.alert)?

You can use this
```javascript
export const MAX_DIALOG_BUTTONS_COUNT = 3;

export default StyleSheet.create({
  buttonBox_vertical: {
    flexDirection: "column",
    height: 46 * MAX_DIALOG_BUTTONS_COUNT
  },
  buttonBox__separator_vertical: {
    width: "100%",
    height: 0.5
  }
});
```

and 
```
<Dialog.Container visible={this.isOpen} {...positioningButtonsStyle}>
```

I did not display more than three buttons as unnecessary, but you can use the height and count of buttons for this.

**Comment by mmazzarolo** [comment]
Available now thanks to #110 !
